### ISSUE TITLE
Lost (in) Translation

### ISSUE THREAD

**Issue by GiacomoSorbi** [Codewars/codewars.com#23]
Not sure if Jake read this already, in case sorry for the repetition, but of all the translation I have done, two apparently seem gone MIA in this [kata](http://www.codewars.com/kata/53db96041f1a7d32dc0004d2/discuss).

Or, rather: they can't be found by either me or the kata creator, but I can read them in my profile page (browsing A LOT of other stuff, but ok) and the kata creator somehow manages to find them among forks or something like that, but can't approve them.

The above link should provide you the full discussion (expand my message). Now, I have no big problem copy&pasting said translations into two new ones (provided this work), but I was wondering if that could hinder your search for the problem, or, worse, do some more damage.

Let us (me and suuuzi) about how to proceed and thanks in advance for your attention :)

**Comment by jhoffner** [comment]
I wasn't able to find the translation either by doing a quick look through the site, I will try to dig into the console and see if I can't find the item directly and get back to you.

**Comment by GiacomoSorbi** [comment]
Not sure if it helps, but now that I read their story, suuuzi apparently
approved, forked, deprecated and finally rejected her (and my) version:
http://www.codewars.com/kumite/550df2b3e051b3eca0000540?sel=550df2b3e051b3eca0000540

http://www.codewars.com/kumite/550f5b8dacc1aa61fd000250?sel=550f5b8dacc1aa61fd000250

Though this is apparently was done after when I couldn't see them anymore
already.

Thanks again for your concern!

**Comment by suuuzi** [comment]
Hi!

Just to update what happened: I remember not see an approve button (I already approved other translations before, so I know where the button should be, I guess). So I try to fork, and after that I click Approve, Save & Publish and nothing happens. I can't find a record of this translation in any place (only the kumite link that I found in my browser history)!

I really don't know if I did something wrong or if something weird happened.

(sorry for the bad english, i'm learning)
